### PR TITLE
Adding PPOM wasm code

### PR DESCRIPTION
**Pull request by jpuri**
Adding wasm code and a `JS` file to interface with it.

For this version was file is included directly, eventually it will be replicated by a npm dependency or some other registry.

### CODE REVIEW

File: src/ppom.js [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+
+  initMemory(imports);
+
+  const { instance, module } = await load(await input, imports);
+
+  return finalizeInit(instance, module);
+}
+
+export { initSync };
+export default init;
```
Comment (jpuri) [inline src/ppom.js:433]: This code is bit of blackbox for me, it has been shared by Blockaid's team. I am not sure what will be best way to include it in our code

I excluded it from lint and tests.

Comment (jpuri) [reply src/ppom.js]: Blockaid team shared that this is an file autogenerated for wasm code and should not be updated manually. I added this information in a comment in the file.

**Review by segun — COMMENTED** [review]
Hi Jyoti, is this the generated code you talked about?

**Comment by jpuri** [comment]
@segun : yep in later PRs this is replaced by npm module.

**Comment by jpuri** [comment]
Review feedbacks are addressed here: https://github.com/MetaMask/ppom-validator/pull/3

**Review by matthewwalsh0 — APPROVED** [review]

**Pull request merged**